src/components/ProfitAndLossSummaries/ProfitAndLossSummaries.tsx

```diff
@@ -1,202 +1,136 @@
-import React, { useContext, useMemo } from 'react'
-import { Scope } from '../../hooks/useProfitAndLoss/useProfitAndLoss'
-import { centsToDollars as formatMoney } from '../../models/Money'
-import { ProfitAndLoss } from '../../types'
-import { LineBaseItem } from '../../types/line_item'
-import {
-  collectExpensesItems,
-  collectRevenueItems,
-} from '../../utils/profitAndLossUtils'
+import React, { useContext, useMemo, type ReactNode } from 'react'
 import { ProfitAndLoss as PNL } from '../ProfitAndLoss'
-import { SkeletonLoader } from '../SkeletonLoader'
-import { MiniChart } from './MiniChart'
-import classNames from 'classnames'
+import {
+  ProfitAndLossSummariesList,
+  ProfitAndLossSummariesListItem,
+} from './internal/ProfitAndLossSummariesList'
+import {
+  ProfitAndLossSummariesMiniChart,
+  toMiniChartData,
+} from './internal/ProfitAndLossSummariesMiniChart'
+import { ProfitAndLossSummariesSummary } from './internal/ProfitAndLossSummariesSummary'
 
 export interface ProfitAndLossSummariesStringOverrides {
   revenueLabel?: string
   expensesLabel?: string
   netProfitLabel?: string
 }
 
-const CHART_PLACEHOLDER = [
-  {
-    name: 'placeholder',
-    display_name: 'placeholder',
-    value: 1,
-    type: 'placeholder',
-    share: 1,
-  },
-]
-
-const buildMiniChartData = (scope: Scope, data?: ProfitAndLoss) => {
-  if (!data) {
-    return CHART_PLACEHOLDER
-  }
-
-  let items: LineBaseItem[] = []
-
-  switch (scope) {
-    case 'revenue':
-      items = collectRevenueItems(data)
-      break
-    default:
-      items = collectExpensesItems(data)
-  }
-
-  if (
-    !items ||
-    items.length === 0 ||
-    !items.find(x => Math.abs(x.value) !== 0)
-  ) {
-    return CHART_PLACEHOLDER
-  }
-
-  return items.slice()
-}
+const SECTION_CLASS_NAME = 'Layer__ProfitAndLossSummaries'
+const SECTION_CLASS_NAMES = `${SECTION_CLASS_NAME} Layer__component`
 
 type ProfitAndLossSummariesProps = {
-  vertical?: boolean
   actionable?: boolean
+  stringOverrides?: ProfitAndLossSummariesStringOverrides
+  chartColorsList?: string[]
+  slots?: {
+    unstable_AdditionalListItems?: [ReactNode]
+  }
+  variants?: {
+    size: 'sm' | 'lg'
```

Comment:
I would imagine this should be an optional property. In the future I'm sure we'll have use cases for a variant type other than size and we'd want them to be independently specifiable, yes?

Reply:
Converted to an optional property.